### DISCUSSION TITLE
cannot install jdtls

### DISCUSSION THREAD

**Discussion by truongle2004** [nvim-java/nvim-java#205] [Q&A] [answered]
![image](https://github.com/nvim-java/nvim-java/assets/119171229/de1c251b-1cc9-41ef-8aa0-852a3ad5a229)
how i can fix that

**Comment by truongle2004** [comment]
how i can change the version of jdtls in your default plugin?

**Reply by s1n7ax** [reply, marked as answer]
https://github.com/nvim-java/nvim-java/wiki/Q-&-A#no_entry-cannot-find-package-xxxxx

**Reply by truongle2004** [reply]
it's working thanks a lot, but there is still one error that I cannot install jdk17, but luckily when I disable auto-install jdk17 in my java it still works.

**Reply by s1n7ax** [reply]
@truongle2004 What is the error you are seeing?

**Reply by truongle2004** [reply]
![image](https://github.com/nvim-java/nvim-java/assets/119171229/b04168a5-6c2d-43ee-811d-9d1bc99003bb)
I am currently using Windows and PowerShell for neovim

**Reply by s1n7ax** [reply]
This should be fixed now. You have to update so the plugins and manually remove jdk using Mason. The next time its installed, the Java path should be correctly set

**Reply by truongle2004** [reply]
![image](https://github.com/nvim-java/nvim-java/assets/119171229/18079e85-af34-446e-a77e-edd30f9352a8)
It works, I removed all the nvim-data files and reinstalled them.
